Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION}
+
+ENV ENV="Local"
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Using `COPY . .` can lead to larger image sizes if unnecessary files are included. Ensure your `.dockerignore` file is set up to exclude these files, which seems to be done correctly.